docker/Dockerfile.dev

```
@@ -1,16 +1,25 @@
+FROM fedora as words
+
+# needed to install words file
+RUN yum -y install words
+
 FROM golang:alpine
 
+# words file needed for generating random words
+COPY --from=words /usr/share/dict/words /usr/share/dict/words
 RUN apk add --update nodejs yarn git util-linux
 
 WORKDIR /app
 COPY . .
 
 RUN go mod download
+RUN cd ui && yarn
+
 # Make sure we don't pollute the main docker-file
 # CompileDaemon allows for hot reloading
 RUN cd /tmp && go get github.com/githubnemo/CompileDaemon
 RUN go build
 
 EXPOSE 8900
-# Want to run yarn watch in the background for hot reloading
-CMD (cd ui && yarn watch &) && CompileDaemon --command="./shoe-project server"
+
+CMD go run ./shoe-project server
```

let's go with exec form if it's a single command, it has better optimizations + works better with signals iirc.

```suggestion
CMD ["go", "run", "./shoe-project", "server"]
```

---

right! looks much cleaner